User-Help/DeletedPostHelp

```diff
@@ -0,0 +1,15 @@
+| URL slug | Category |
+|:--------:|:--------:|
+| articles | Site Information |
+
+## Why are posts deleted? 
+
+Posts are deleted for various reasons. They may be deleted either by a moderator or by the original poster.
+
+A question may be deleted because it is off topic for a particular community or category. That doesn't mean it's a bad question, just that it's not in the right place. For example, a question about musical instruments would likely be removed from a gardening community. Sometimes it's more subtle; a question about how to make flower pots _could_ be considered gardening, but the community might want to keep posts more focused. Similarly, an opinion based question (What is the prettiest flower?) might not be on topic in some communities. Always check the community description and any linked help before posting to make sure your question will be welcomed! 
+
+An answer may be put on hold or removed for similar reasons; it's off-topic, not answering the question, etc. 
```

Comment:
Sorry I must have missed this before -- answers can't be put on hold, only questions.  I would say "An answer may be deleted for similar reasons..." -- deleted rather than removed for parallelism with the previous paragraph (so people don't wonder if there's a difference), and not talking about closure.